[2024-12-03T16:28:31.712+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-03T16:28:31.780+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: streaming_dag_v3.stream_to_kafka_topic scheduled__2024-12-03T15:30:00+00:00 [queued]>
[2024-12-03T16:28:31.810+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: streaming_dag_v3.stream_to_kafka_topic scheduled__2024-12-03T15:30:00+00:00 [queued]>
[2024-12-03T16:28:31.812+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-03T16:28:31.845+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): stream_to_kafka_topic> on 2024-12-03 15:30:00+00:00
[2024-12-03T16:28:31.856+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'streaming_dag_v3', 'stream_to_kafka_topic', 'scheduled__2024-12-03T15:30:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/stream_to_spark.py', '--cfg-path', '/tmp/tmpo11q46ya']
[2024-12-03T16:28:31.854+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=333) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-03T16:28:31.862+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask stream_to_kafka_topic
[2024-12-03T16:28:31.863+0000] {standard_task_runner.py:72} INFO - Started process 334 to run task
[2024-12-03T16:28:31.948+0000] {task_command.py:467} INFO - Running <TaskInstance: streaming_dag_v3.stream_to_kafka_topic scheduled__2024-12-03T15:30:00+00:00 [running]> on host a5c8fad7717c
[2024-12-03T16:28:32.121+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='neiko' AIRFLOW_CTX_DAG_ID='streaming_dag_v3' AIRFLOW_CTX_TASK_ID='stream_to_kafka_topic' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T15:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-03T15:30:00+00:00'
[2024-12-03T16:28:32.124+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-03T16:33:33.331+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-12-03T16:38:44.410+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-12-03T16:43:55.494+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-12-03T16:49:06.575+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-12-03T16:54:17.656+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-12-03T16:59:28.744+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-12-03T17:04:39.827+0000] {logging_mixin.py:190} INFO - Message delivery failed: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
